test/ex_doc/formatter/html_test.exs

```diff
@@ -121,8 +121,9 @@ defmodule ExDoc.Formatter.HTMLTest do
     assert File.regular?("#{output_dir()}/another_dir/CompiledWithDocs.html")
     assert File.regular?("#{output_dir()}/another_dir/RandomError.html")
 
-    assert "#{output_dir()}/another_dir/dist/app-*.css" |> Path.wildcard |> File.regular?
-    assert "#{output_dir()}/another_dir/dist/app-*.js" |> Path.wildcard |> File.regular?
+    for file <- Path.wildcard("#{output_dir()}/another_dir/dist/app-*.{js,css}") do
+      assert File.regular?(file) == true
```

Comment:
Even that should be rewritten to 

```elixir
assert "abcd" =~ ~R/.../
refute ...
```

so that ExUnit can potentially produce a diff in case of failure. The only reason to use `== true` (or `false`) as far as I know is when the function can return something else than a boolean and you want to assert that the specific case you're testing returns a boolean (for example `assert foo` would pass if `foo` was anything non-`nil` and non-`false`).